packages/suite/src/components/suite/DropZone/index.tsx

```diff
@@ -2,9 +2,14 @@ import React, { useRef, useCallback, useMemo, useState } from 'react';
 import styled from 'styled-components';
 import { P, Icon } from '@trezor/components';
 import { Translation } from '@suite-components';
+import { IconType } from '@trezor/components/src/support/types';
 
 interface Props {
-    accept: 'text/csv' | 'image/*' | 'application/octet-stream';
+    // 'accept' attribute for underlying HTML file input
+    accept?: string;
```

Comment:
Maybe an array of acceptable mime types?
For binary `['application/octet-stream', 'application/mac-binary']`

Reply:
The thing is that

1. now the 'caller' of the component decides what is and what isn't acceptable, and
2. the 'accept' attribute can contain not only mimetypes, but file extensions as well.